styles/components/_hero.scss

```diff
@@ -0,0 +1,47 @@
+.hero {
+  background: $tundora 100% / cover;
+  background-position: center;
+
+  .container {
+    position: relative;
+    min-height: 250px;
+  }
+
+  h1 {
+    @include span (12);
+    font-family: $font-body;
+    font-style: italic;
+    font-weight: 900;
+    position: absolute;
+    bottom: 1rem;
+    color: $white;
+  }
+}
+
+
+// Image selections based on page-SLUG
+.page-about {
+  .hero {
+    background-image: url(../../images/hero/placeholder.jpg);
+  }
+}
+.page-contact {
+  .hero {
+    background-image: url(../../images/hero/placeholder.jpg);
+  }
+}
+.category-civil-projects {  // This will be changed when category names are fixed
```

Comment:
Sure, so do that when you have one. Are you worried you'll forget how unless these lines exist?